src/components/Forms/SelfScan/index.js

```diff
@@ -301,15 +287,15 @@ const CountSignaturesForm = ({
 
   return (
     <>
-      {isAuthenticated && (
+      {userId && (
```

Comment:
Shows the login state when userId is present, not just authenticated

Reply:
This works on this page, all data needed for the page is available via user api or the analytics/signatures api